src/_includes/certified/common.njk

```diff
@@ -0,0 +1,29 @@
+    <div class="px-2 flex items-center gap-12">
```

Comment:
`common.njk` I think would be better named like the actual function in the page. It's not quite a header, but maybe call it something else?